src/test/java/pl/prawko/prawko_server/repository/RoleRepositoryTest.java

```diff
@@ -0,0 +1,35 @@
+package pl.prawko.prawko_server.repository;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import pl.prawko.prawko_server.model.Role;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Search in the whole project for `org.junit.jupiter.api.Assertions` and get rid of it from everywhere